src/Model.cpp

```diff
@@ -27,7 +27,8 @@ extern Statistics stats;
 void Model::runScenarios(ScenariosList &scenarios, Population &popln,
                          Vector &vectors, Worm &worms, int replicates,
                          double timestep, int index, int outputEndgame,
-                         int outputEndgameDate, int reduceImpViaXml,
+                         int outputEndgameDate, bool outputNTDMC,
```

Comment:
💚 This is good to be using boolean rather than an int with the others, makes it clear what the intention of the variable is, and prevents passing muddling up the two parameters.

One neat way to encode this common pair (of some data, and a boolean flag to say whether to use the data) is [`std::optional`](https://en.cppreference.com/w/cpp/utility/optional):

```suggestion
                         int outputEndgameDate, std::optional<int> outputNTDMCDate,
```

And then when you use it you can do:

```c++
if(outputNTDMCDate.hasValue()){
  outputNTDMCDateInYears = (outputNTDMCDate.value - BASEYEAR) * 12;
}
```
But no need to do that for now.